src/main/java/org/eclipse/pass/schema/SchemaServlet.java

```diff
@@ -32,12 +41,33 @@
 @WebServlet("/schemaservice")
 public class SchemaServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
+    private List<String> repository_list;
+    private String next;
 
     /**
      * Servlet constructor.
      */
     public SchemaServlet() {
-        // TODO: Not implemented yet
+        repository_list = new ArrayList<String>();
+    }
+
+    public List<String> readText(BufferedReader r) throws IOException {
```

Comment:
You should generally not have public methods on a Servlet class. This could be protected, but even that should be avoided in most cases.